spec/classes.md

```diff
@@ -2873,9 +2873,18 @@ the underlying `TextWriter` for the output device is created. But if the applica
 
 ### Automatically implemented properties
 
-An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern property with semicolon-only accessor bodies. Auto-properties must have a get accessor and can optionally have a set accessor.
+An automatically implemented property (or ***auto-property*** for short), is a non-abstract non-extern property with either or both:
 
-When a property is specified as an automatically implemented property, a hidden backing field is automatically available for the property, and the accessors are implemented to read from and write to that backing field. If the auto-property has no set accessor, the backing field is considered `readonly` ([Readonly fields](classes.md#readonly-fields)). Just like a `readonly` field, a getter-only auto-property can also be assigned to in the body of a constructor of the enclosing class. Such an assignment assigns directly to the readonly backing field of the property.
+1. an accessor with semicolon-only body
+2. usage of the `field` contextual keyword ([Keywords](lexical-structure.md#keywords)) within the accessors or expression body of the property. The `field` identifier is only considered the `field` keyword when there is no existing symbol named `field` in scope at that location.
+
+Auto-properties must have a get accessor and can optionally have a set accessor. Because an expression-bodied property defines a get accessor, if the expression-bodied property refers to the `field` keyword, it satisfies the requirements and is considered an auto-property.
+
+When a property is specified as an auto-property, a hidden, unnamed, backing field is automatically available for the property, and any semicolon-only accessors are implemented to read from and write to that backing field. The backing field can be referenced directly using the `field` keyword within all accessors and within the property expression body. Because the field is unnamed, it cannot be used in a `nameof` expression.
```

Comment:
```suggestion
When a property is specified as an auto-property, a hidden, unnamed backing field is automatically available for the property, and any semicolon-only accessors are implemented to read from (if a `get` accessor) or write to (if a `set` accessor) that backing field. The backing field can be referenced directly using the `field` keyword within all accessors and within the property expression body. Because the field is unnamed, it cannot be used in a `nameof` expression.
```

Reply:
(I know those parens are not in the original, but what the original says is kinda ambiguous at the moment).

Reply:
Yeah, i'm ok with keeping as is given the original.  We can see what LDM thinks on this.   but i think it's fairly clear contextually :)